services/navigation/data/nav_cfg.json

```diff
@@ -13,13 +13,9 @@
     "services":
     [{
         "name":"test_navigation",
-        "api":"rdk:service:navigation",
-        "model":"rdk:builtin:builtin",
+        "type": "navigation",
         "attributes":{
             "base":"test_base",
-            "degs_per_sec":1,
-            "meters_per_sec":1,
-            "motion_service":"builtin",
```

Comment:
This is changing in a couple json files, why is that?

Reply:
this is me trying to get the cypress test to pass, I can change it back.